packages/flutter/test/widgets/spell_check_test.dart

```diff
@@ -17,9 +17,9 @@ void main() {
     misspelledTextStyle = TextField.materialMisspelledTextStyle;
   });
 
-  test(
-      'buildTextSpanWithSpellCheckSuggestions ignores composing region when composing region out of range',
-      () {
+  testWidgets(
+  'buildTextSpanWithSpellCheckSuggestions ignores composing region when composing region out of range',
+      (WidgetTester tester) async {
```

Comment:
Why make all of these `testWidgets` instead of `test`?  I think if it doesn't use `tester` it should just be a `test`.

Reply:
I did this to use `TargetPlatformVariant`

Reply:
Ah I didn't realize you couldn't use it with `test`!  Sounds good then.